x-pack/plugin/rollup/src/main/java/org/elasticsearch/xpack/downsample/AbstractRollupFieldProducer.java

```diff
@@ -34,14 +40,14 @@ public String name() {
     }
 
     /**
-     * @return the value of the field.
+     * Resets the collected value to the specific subclass reset value.
```

Comment:
It might be more descriptive to say "an empty value" - or something like that. "empty" being a fairly vague concept. Or maybe `zero-like value`. I dunno.